Essentials/src/messages.properties

```diff
@@ -625,3 +625,64 @@ year=year
+radiusTooBig=\u00a74Radius is too big\! Maximum radius is {0}.
+isIpBanned=\u00a76IP \u00a7c{0} \u00a76is banned.
+mobDataList=\u00a76Valid mob data\:\u00a7r {0}
+vanish=\u00a76Vanish for {0}\u00a76\: {1}
+noLocationFound=\u00a74No valid location found.
+coordsKeyword={0}, {1}, {2}
+banExemptOffline=\u00a74You may not ban offline players.
+tempbanExemptOffline=\u00a74You may not tempban offline players.
+mayNotJailOffline=\u00a74You may not jail offline players.
+muteExemptOffline=\u00a74You may not mute offline players.
+ignoreExempt=\u00a74You may not ignore that player.
+unsafeTeleportDestination=\u00a74The teleport destination is unsafe and teleport-safety is disabled.
+noMetaJson=JSON Metadata is not supported in this version of Bukkit.
+maxMoney=\u00a74This transaction would exceed the balance limit for this account.
+skullChanged=\u00a76Skull changed to \u00a7c{0}\u00a76.
+alphaNames=\u00a74Player names can only contain letters, numbers and underscores.
+givenSkull=\u00a76You have been given the skull of \u00a7c{0}\u00a76.
+noPermissionSkull=\u00a74You do not have permission to modify that skull.
+teleportInvalidLocation=Value of coordinates cannot be over 30000000
+invalidSkull=\u00a74Please hold a player skull.
+weatherInvalidWorld=World named {0} not found\!
+gameModeInvalid=\u00a74You need to specify a valid player/mode.
+mailTooLong=\u00a74Mail message too long. Try to keep it below 1000 characters.
+mailDelay=Too many mails have been sent within the last minute. Maximum\: {0}
+seenAccounts=\u00a76Player has also been known as\:\u00a7c {0}
+unableToSpawnItem=\u00a74Cannot spawn \u00a7c{0}\u00a74, this is not a spawnable item.
+itemsConverted=\u00a76Converted all items into blocks.
+itemsNotConverted=\u00a74You have no items that can be converted into blocks.
+mailSentTo=\u00a7c{0}\u00a76 has been sent the following mail\:
+mailMessage={0}
+whoisTempBanned=\u00a76 - Ban expires:\u00a7r {0}
+playerTempBanned=\u00a76Player \u00a7c{0}\u00a76 temporarily banned \u00a7c{1}\u00a76 for \u00a7c{2}\u00a76: \u00a7c{3}\u00a76.
+mailFormat=\u00a76[\u00a7r{0}\u00a76] \u00a7r{1}
+cantGamemode=\u00a74You do not have permission to change to gamemode {0}
+createKit=\u00a74/createkit <kitname> <delay>
+createdKit=\u00a76Created kit \u00a7c{0} \u00a76with \u00a7c{1} \u00a76entries and delay \u00a7c{2}
+spectator=spectator
+kitContains=\u00a76Kit \u00a7c{0} \u00a76contains:
+kitItem=\u00a76- \u00a7f{0}
+invalidBanner=\u00a74Invalid banner syntax.
+msgDisabled=\u00a76Receiving messages \u00a7cdisabled\u00a76.
+msgDisabledFor=\u00a76Receiving messages \u00a7cdisabled \u00a76for \u00a7c{0}\u00a76.
+msgEnabled=\u00a76Receiving messages \u00a7cenabled\u00a76.
+msgEnabledFor=\u00a76Receiving messages \u00a7cenabled \u00a76for \u00a7c{0}\u00a76.
+msgIgnore=\u00a7c{0} \u00a74has messages disabled.
+minimumPayAmount=\u00a7cThe minimum amount you can pay is {0}.
+itemnameInvalidItem=\u00a7cYou need to hold a renamable item to rename it.
```

Comment:
The only material that this command will refuse to rename is air (an empty hand). I maintain that my earlier suggestion is more logical, as an empty hand is not an item to the majority of players.

Reply:
Understood - Apologies, I think I mis-read your suggestion on the original PR as a question :)
I will make this change as requested.